logging/src/main/java/org/jboss/as/logging/handlers/HandlerOperations.java

```diff
@@ -650,22 +651,19 @@ private static void handleProperty(final AttributeDefinition attribute, final Op
             if (valueNode.isDefined()) {
                 final String resolvedValue = valueNode.asString();
                 configuration.setFormatterName(resolvedValue);
-                // If the formatter was previously defined by the formatter attribute, remove the formatter
-                if (logContextConfiguration.getFormatterNames().contains(handlerName)) {
-                    logContextConfiguration.removeFormatterConfiguration(handlerName);
-                }
```

Comment:
We still need this to remove the `handlerName + PatternFormatterResourceDefinition.DEFAULT_FORMATTER_SUFFIX`.

Reply:
That is taken care of [here](https://github.com/wildfly/wildfly-core/blob/8839ff61c041f930b4128d1d7351d779a8981159/logging/src/main/java/org/jboss/as/logging/handlers/HandlerOperations.java#L632), isn't it? The test is passing.

Reply:
Only if the `formatter` attribute is processed. So for example if you were to add a handler then write a `named-formatter` attribute, that remove would not be hit.

For example:
```
/subsystem=logging/console-handler=my-console:add
/subsystem=logging/console-handler=my-console:write-attribute(name=named-formatter, value=COLOR-PATTERN)
```